tests/fixtures/opensuse-leap-15.5/efibootmgr.out

```diff
@@ -0,0 +1,17 @@
+BootCurrent: 0004
+Timeout: 0 seconds
+BootOrder: 0004,0001,0007,0002,0003,0009,0000,0005,0006,0008,000A,000B
+Boot0000* debian	HD(1,GPT,66b2a325-3129-45d3-84dd-c1a4e4bd846b,0x800,0x100000)/File(\EFI\debian\shimx64.efi)
+Boot0001* RecoveryBoot	HD(3,GPT,a75ab570-de77-504c-be97-35faa629f1f3,0x5dc800,0x1f4000)/File(EFI\boot\bootx64.efi)
+Boot0002  Embedded NIC 1 Port 1 Partition 1	VenHw(3a191845-5f86-4e78-8fce-c4cff59f9daa)
+Boot0003  Embedded NIC 2 Port 1 Partition 1	VenHw(d227c733-f75f-4341-b749-4d1759ec8538)
+Boot0004* PrimaryBoot	HD(1,GPT,faf946be-a174-478f-82ea-d90aa476fb91,0x800,0xfa000)/File(EFI\boot\bootx64.efi)
+Boot0005* EFI Fixed Disk Boot Device 3	PciRoot(0x1)/Pci(0x4,0x0)/Pci(0x0,0x0)/NVMe(0x1,8C-E3-8E-E2-0A-D4-0F-01)/HD(1,GPT,c2336d30-9bde-4a0b-b9be-145d6958d5af,0x1000,0x80001)
+Boot0006* EFI Fixed Disk Boot Device 4	PciRoot(0x11)/Pci(0x1,0x0)/Pci(0x0,0x0)/NVMe(0x1,00-25-38-95-21-01-B8-C1)/HD(1,GPT,1fbfeaec-d78d-4d8c-93ae-77bba6c09d3f,0x1000,0x80001)
+Boot0007* BackendBoot	PciRoot(0x11)/Pci(0x3,0x0)/Pci(0x0,0x0)/NVMe(0x1,00-25-38-95-21-01-B5-B6)/HD(1,GPT,b95c4d58-849f-40fa-bf2b-4f0b8a0aa8bd,0x1000,0x80001)
+Boot0008* EFI Fixed Disk Boot Device 6	PciRoot(0x11)/Pci(0x6,0x0)/Pci(0x0,0x0)/NVMe(0x1,8C-E3-8E-E2-0A-D3-90-01)/HD(1,GPT,f543ab7b-9d55-4e57-8afe-bdb907428f15,0x1000,0x80001)
+Boot0009* EFI Fixed Disk Boot Device 7	PciRoot(0x12)/Pci(0x6,0x0)/Pci(0x0,0x0)/NVMe(0x1,00-25-38-95-21-01-B6-08)/HD(1,GPT,db9522dc-062b-4efa-84b5-6956cea7b5a6,0x1000,0x80001)
+Boot000A* EFI Fixed Disk Boot Device 8	PciRoot(0x12)/Pci(0x8,0x0)/Pci(0x0,0x0)/NVMe(0x1,00-25-38-95-21-01-B6-0D)/HD(1,GPT,4a9e1d40-b853-44a0-8d26-a4df6f112dc9,0x1000,0x80001)
+Boot000B* EFI Fixed Disk Boot Device 9	PciRoot(0x13)/Pci(0x5,0x0)/Pci(0x0,0x0)/NVMe(0x1,8C-E3-8E-E2-0A-D3-8E-01)/HD(1,GPT,5a80e168-bac7-4967-9507-e23a432bae18,0x1000,0x80001)
+MirroredPercentageAbove4G: 0.00
+MirrorMemoryBelow4GB: false
```

Comment:
Should we add these fields to the output dict? If so, the values should also be converted to float and bool in the `_process()` function.

Reply:
Not sure if any one would interest in these fields, adding to the output anyway.